L1Trigger/TrackFindingTracklet/src/PurgeDuplicate.cc

```diff
@@ -181,38 +187,70 @@ void PurgeDuplicate::execute(std::vector<Track>& outputtracks, unsigned int iSec
         if (inputtracklets_.empty())
           continue;
         const unsigned int numStublists = inputstublists_.size();
+        std::vector<int> seedRankIdx(numStublists);
```

Comment:
Add comment that you're ordering the tracks by seed rank.

Reply:
Comment has been included.